Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,250 @@
+```
+The RSAKeys class' function .save() will return a String representation of the key in Base64 format. This mime type is based on the hexadecimal notation used in the PEM format readable by OpenSSL.
+
+```scala
+@ val basePublic = RSAKeys.save(pub)
+
+basePublic: String = """MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAk3rczXIryYKu1IR4crgeNriQvKExZnFL
+wr7+TYVHtiGOzS2h6wIBmKTqAOTbZ1fH3ac47I24s78hHTo6F+GWogNbxMedBtilgUh9n0noY3Rx
+KxDvUA36JCogyrUpEeJjbJ2Zsh/pdo7yOBmJol3IsLekZTEkmqwnxLirRRoZ1fvfpfeLDerJd4x/
++Hz2EGrkpkM0Zr6yHfEmW/H8mrnMgNev+MxND2euKGR+QEjajfdTSTuN5qjglhQWtLN/cBKQfS51
+YDS4qExuSVyPHYH2mEOuUTeVcdg7OOHAigjDdI73XtfssBbQyEJrMMjFBgoI+H9nZLDsFGZ81vDa
+oSRBVwIDAQAB"""
+```
+
+All keys can be converted from a PEM formatted String back to decimal format. The readPublicKeyFrom() function takes a String in Base64 notation and returns the original value of the key. This means that a public key, saved to a file in PEM format can always be converted back to it's original format by the user at a later point in time.
+
+```scala
+@ val publ = RSAKeys.readPublicKeyFrom(basePublic)
+
+publ: scala.util.Try[java.security.PublicKey] = Success(
+  Sun RSA public key, 2048 bits
+  modulus: 18617616591683306896114159596818930151392321021655743092554571476906272216221226120621265951005438129779697999635487320523329319543618461573656972895570742121788481588357965539288547158041270238222755734178260848243117538931191184092136701393910768801017892322044665470799614169981926296276068464740129396666480935344897491410772746293264767223181696074228079861695236831569819678997680346803798366156295040872828579274684209593397084757356046266872087618604860475724560124477904326733099073845912710038077198861072318221134249054814455997792289530305256172821638522445655947916737229103835394051258962680413560258903
+  public exponent: 65537
+)
+```
+The .readPublicKeyFrom() function returns a Try of Public/Private key depending on whether the String given can be parsed back to some valid key. If the operation is successful the value of the original key is assigned to the new value in BigInt format.
+
+The user can then save his keys on his local filesystem by using the following ammonite commands:
+
+```scala
+write(wd/"publicKey.pem", RSAKeys.save(pub))
+write(wd/"privateKey.pem", RSAKeys.save(priv))
+```
+This will save the contents of the key in String format within the file found by following the specified path in .pem files.
+
+### Saving a public Key to RDF
+
+A user can also transform his public keys to an RDF. In order to do this one must first import multiple files in order to resolve the required dependencies.
```

Comment:
"In order ..."
We will use the banana binder library to show how this process can be done automatically and elegantly.